docs/library/jesd204/index.rst

```diff
@@ -495,7 +494,6 @@ HDL Example Projects
 
 - :ref:`AD6676EVB Reference Design <ad6676evb>` on:
 
-  - :git-hdl:`AMD Xilinx VC707 <projects/ad6676evb/vc707>`
```

Comment:
This can be changed to state it is retired and to point to the latest release branch which has it

Reply:
done

Reply:
@sarpadi to check this last comment as well. It should be **ad6676evb**